### ISSUE TITLE
Centered BitmapText is rendered blurry

### ISSUE THREAD

**Issue by slyh80** [jMonkeyEngine/jmonkeyengine#358]
Class 'BitmapText' renders centered (`Align.Center`) texts blurry in GUI.

What it actually looks like:
https://www.dropbox.com/s/wqz4yo2gcdgtfum/centered-text-ugly2.png

What it should look like:
https://www.dropbox.com/s/bn46tgdg1ygvthl/centered-text-nice2.png

Code to reproduce:

```
    @Override
    public void simpleInitApp()
    {
        BitmapFont font = getAssetManager().loadFont("Interface/Fonts/Console.fnt");
        BitmapText text = font.createLabel("This is a centered test text that does not look nice");
        text.setBox(new Rectangle(250, 240, 110, 100));
        text.setAlignment(Align.Center);

        getGuiNode().attachChild(text);
    }
```

**Comment by stephengold** [comment]
This looks like a one-line fix.

**Comment by pspeed42** [comment]
No.  This fix assumes that bitmap text is being rendered on local pixel boundaries.  For a variety of reasons this may not be the case.

For example, if you alter that test to setSize(0.1) then I suspect you will have text centered in giant character-wide leaps.  This fix needs to be reverted and the issue closed as unfixable: "not a bug"

**Comment by stephengold** [comment]
Thank you for the feedback, Paul. I've reverted the fix but left the issue open for easy reference.
